src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -212,8 +212,10 @@ class DataBox<tmpl::list<Tags...>> : private detail::Item<Tags>... {
    * \note the default constructor is only used for serialization
    */
   DataBox() = default;
-  DataBox(DataBox&& rhs) = default;
-  DataBox& operator=(DataBox&& rhs) {
+  constexpr DataBox(DataBox&& rhs) : detail::Item<Tags>(std::move(rhs))... {
```

Comment:
can we add a test that would have failed without this change?

Reply:
I've modified Test_DataBox to check this.  Didn't make it a fixup because these are the only changes to that file in this PR.